flink-connectors/flink-jdbc/src/main/java/org/apache/flink/api/java/io/jdbc/dialect/JDBCDialect.java

```diff
@@ -139,4 +146,46 @@ default String getSelectFromStatement(String tableName, String[] selectFields, S
 		return "SELECT " + selectExpressions + " FROM " +
 				quoteIdentifier(tableName) + (conditionFields.length > 0 ? " WHERE " + fieldExpressions : "");
 	}
+
+	/**
+	 * Get create table statement.
+	 */
+	default String getCreateTableStatement(String tableName, String[] fieldNames,
```

Comment:
According flink code style: every parameter should be a single line if you want to break one line.